<launch>
    <node pkg="spirecv_ros" type="rtsp_forward" name="rtsp_forward" output="screen" respawn="true">
        <rosparam file="$(find spirecv_ros)/config/rtsp_forward_config.yaml" command="load"/>
    </node>
</launch>
